core/lib/multivm/src/versions/vm_fast/tests/code_oracle.rs

```diff
@@ -91,6 +93,16 @@ fn test_code_oracle() {
     assert!(!result.result.is_failed(), "Transaction wasn't successful");
 }
 
+fn find_code_oracle_cost_log(
+    precompiles_contract_address: Address,
+    logs: &[StorageLogWithPreviousValue],
+) -> &StorageLogWithPreviousValue {
+    let log = logs.iter().find(|log| {
+        *log.log.key.address() == precompiles_contract_address && *log.log.key.key() == H256::zero()
```

Comment:
```suggestion
        *log.log.key.address() == precompiles_contract_address && log.log.key.key().is_zero()
```

[`is_zero()`](https://docs.rs/primitive-types/latest/primitive_types/struct.H256.html#method.is_zero)